HeterogeneousCore/AlpakaServices/plugins/BuildFile.xml

```diff
@@ -0,0 +1,18 @@
+<!-- alpaka-based portable plugins -->
+<library file="alpaka/*.cc" name="HeterogeneousCoreAlpakaServicesPlugins">
+  <use name="alpaka"/>
+  <use name="FWCore/MessageLogger"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/PluginManager"/>
+  <use name="FWCore/ServiceRegistry"/>
+  <use name="HeterogeneousCore/AlpakaInterface"/>
+  <!--
+  The dependency on "HeterogeneousCore/AlpakaServices" should automatically expand to include
+  the host-only library (if it exists) and corresponding Alpaka libraries (if they exist)
+  -->
+  <!-- <use name="HeterogeneousCore/AlpakaServices"/> -->
+  <use name="HeterogeneousCore/AlpakaServices/alpaka/cuda" for="alpaka/cuda"/>
+  <use name="HeterogeneousCore/AlpakaServices/alpaka/serial" for="alpaka/serial"/>
```

Comment:
@fwyzard , alpaka build rules ( already available in 12.5.X IBs) now automatically include host-only library (if it exists) and corresponding Alpaka libraries (if they exist). So `<use name="HeterogeneousCore/AlpakaServices"/>` should work now

Reply:
Thanks, I've updated the BuildFile.xml .